enterprise/backup/src/main/java/org/neo4j/backup/OnlineBackup.java

```diff
@@ -302,7 +302,7 @@ public OnlineBackup full( String targetDirectory, boolean verification, Config t
     public OnlineBackup incremental( String targetDirectory )
     {
         outcome = new BackupService().doIncrementalBackup( hostNameOrIp, port, new File( targetDirectory ),
-                timeoutMillis, defaultConfig() );
+                getConsistencyCheck( true ), timeoutMillis, defaultConfig() );
```

Comment:
agree with @spacecowboy arguments, switch to not do consistency check by default